launch/tier4_simulator_launch/launch/simulator.launch.xml

```diff
@@ -103,6 +104,16 @@
     </include>
   </group>
 
+  <!-- Planning Evaluator -->
+  <group if="$(var launch_planning_evaluator)">
+    <let name="launch_diagnostic_converter" value="true" if="$(var scenario_simulation)"/>
+    <let name="launch_diagnostic_converter" value="false" unless="$(var scenario_simulation)"/>
```

Comment:
you can just use the `scenario_simulation` var directly for the `launch_diagnostic_converter` value.